torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1676,6 +1676,38 @@ def sample_generator():
 
     return list(sample_generator())
 
+def sample_inputs_copysign(op_info, device, dtype, requires_grad):
+    def _make_case(case):
+        if isinstance(case, tuple):
+            return make_tensor(case, device, dtype,
+                               low=None, high=None,
+                               requires_grad=requires_grad)
+        else:
+            return case
+
+    cases = (
+        # no broadcast
+        ((S, S, S), (S, S, S)),
+        # broadcast rhs
+        ((S, S, S), (S, S)),
+
+        # Broadcasts `self` : Issue with inplace-variants
+        # Reference: https://github.com/pytorch/pytorch/issues/50747
+        # We should enable the below lines once the issue is resolved.
+        # ((S, S), (S, S, S)),
+        # ((S, 1, S), (M, S)),
+
+        # scalar
+        ((S, S), 3.14),
+        # scalar positive zero
+        ((S, S), 0.0),
+        # scalar negative zero
+        ((S, S), -0.0),
+    )
+
+    return [SampleInput(_make_case(lhs), args=(_make_case(rhs),))
+            for lhs, rhs in cases]
+
```

Comment:
Besides using `_make_case(lhs)` to generate input tensor of `SampleInput`, the `_make_case(rhs)` is used for `args` as well.
The `rhs` corresponds to the second item of each tuple element.
Hence its values are 3.14, 0.0, and -0.0 for the corresponding scalar cases.